src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DocumentPresentation/UriPresentationParams.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using MediatR;
+using Newtonsoft.Json;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.DocumentPresentation
+{
+    /// <summary>
+    /// Class representing the parameters sent for a textDocument/_vs_textPresentation request.
+    /// </summary>
+    internal class UriPresentationParams : ITextDocumentIdentifierParams, IRequest<WorkspaceEdit?>, IBaseRequest
+    {
+        /// <summary>
+        /// Gets or sets the identifier for the text document to be operate on.
+        /// </summary>
+        [JsonProperty("_vs_textDocument")]
+        public TextDocumentIdentifier TextDocument { get; set; }
+
+        /// <summary>
+        /// Gets or sets the range.
+        /// </summary>
+        [JsonProperty("_vs_range")]
+        public Range Range { get; set; }
+
+        /// <summary>
+        /// Gets or sets the URI values. Valid for DropKind.Uris.
```

Comment:
>Valid for DropKind.Uris. [](http://example.com/codeflow?start=41&length=24)

Hrm, I'm guessing you copied this from somewhere I shouldn't have kept it.

Reply:
Yeah, straight from `VSInternalUriPresentationParams` :D